app/src/main/res/values/strings.xml

```diff
@@ -114,6 +114,8 @@
     <string name="txtStores">Stores :</string>
     <string name="txtCountries">Countries where sold :</string>
     <string name="txtIngredients">Ingredients list :</string>
+    <string name="Donate">Donate</string>
+    <string name="Donation">Click here to donate for Open Food facts</string>
```

Comment:
"Click here to donate for Open Food facts" 
Can you change the string to "Click here to donate for Open Food Facts" ?

Reply:
It should actually be "Click here to donate to Open Food Facts"